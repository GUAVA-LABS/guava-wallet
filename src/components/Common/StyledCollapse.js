import styled from 'styled-components';
import { Collapse } from 'antd';

export const StyledCollapse = styled(Collapse)`
    background: #fbfcfd !important;
    border: 1px solid #eaedf3 !important;

    .ant-collapse-content {
        border: 1px solid #eaedf3;
        border-top: none;
    }

    .ant-collapse-item {
        border-bottom: none !important;
    }

    * {
        color: rgb(62, 63, 66) !important;
    }
`;
