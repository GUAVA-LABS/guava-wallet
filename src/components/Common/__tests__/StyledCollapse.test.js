import React from 'react';
import renderer from 'react-test-renderer';
import { StyledCollapse } from '../StyledCollapse';

test('Render StyledCollapse component', () => {
    const component = renderer.create(<StyledCollapse />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
