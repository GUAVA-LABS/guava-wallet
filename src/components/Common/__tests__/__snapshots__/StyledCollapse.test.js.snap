// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render StyledCollapse component 1`] = `
<div
  className="ant-collapse ant-collapse-icon-position-left sc-bdVaJa jkIKTL"
  role={null}
/>
`;
